app/assets/stylesheets/retros.scss

```diff
@@ -50,24 +50,35 @@ html {
       }
     }
 
-    #Boards {
-      align-items: center;
+    #sections {
       display: flex;
-      flex-direction: column;
+      flex-direction: row;
+      justify-content: center;
 
-      #discuss {
+      .section {
         border-width: 1px;
-        width: 200px;
-        height: 300px;
+        width: 500px;
+        height: 400px;
         margin-top: 10%;
         display: none;
+        margin-left: 20px;
+        margin-right: 20px;
       }
     }
 
-    legend {
+    #mention > legend {
```

Comment:
I think you could reuse the styles for #mention and #discuss legends, since thay are basically the same, and simply add the extra properties to the discuss legend. What do you think?